### ISSUE TITLE
Error in creating iOS build using fluttertoast: ^7.0.3

### ISSUE THREAD

**Issue by deepak786** [ponnamkarthik/FlutterToast#218]
This is the error:

```In file included from
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/FluttertoastPlugin.m:2:
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/UIView+Toast.h:328:19: warning: empty paragraph passed to '@param'
    command [-Wdocumentation]
     @param sharedStyle
     ~~~~~~~~~~~~~~~~~^
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/UIView+Toast.h:343:25: warning: empty paragraph passed to '@param'
    command [-Wdocumentation]
     @param allowTapToDismiss
     ~~~~~~~~~~~~~~~~~~~~~~~^
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/UIView+Toast.h:343:9: warning: parameter 'allowTapToDismiss' not
    found in the function declaration [-Wdocumentation]
     @param allowTapToDismiss
            ^~~~~~~~~~~~~~~~~
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/UIView+Toast.h:343:9: note: did you mean 'tapToDismissEnabled'?
     @param allowTapToDismiss
            ^~~~~~~~~~~~~~~~~
            tapToDismissEnabled
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/UIView+Toast.h:362:20: warning: empty paragraph passed to '@param'
    command [-Wdocumentation]
     @param queueEnabled
     ~~~~~~~~~~~~~~~~~~^
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/FluttertoastPlugin.m:20:23: warning: unused variable 'viewController'
    [-Wunused-variable]
        UIViewController *viewController =
                          ^
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/FluttertoastPlugin.m:76:22: error: assigning to 'NSNumber *__strong'
    from incompatible type 'double'
                fontSize = 16.0;
                         ^ ~~~~
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/FluttertoastPlugin.m:97:21: warning: unused variable 'topPadding'
    [-Wunused-variable]
                CGFloat topPadding = window.safeAreaInsets.top;
                        ^
    /Library/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/C
    lasses/FluttertoastPlugin.m:98:21: warning: unused variable 'bottomPadding'
    [-Wunused-variable]
                CGFloat bottomPadding = window.safeAreaInsets.bottom;

**Comment by polilluminato** [comment]
I've got the exact same error with `fluttertoast: ^7.0.3`

```dart
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    In file included from /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:2:
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:328:19: warning: empty paragraph passed to '@param' command [-Wdocumentation]
     @param sharedStyle
     ~~~~~~~~~~~~~~~~~^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:343:25: warning: empty paragraph passed to '@param' command [-Wdocumentation]
     @param allowTapToDismiss
     ~~~~~~~~~~~~~~~~~~~~~~~^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:343:9: warning: parameter 'allowTapToDismiss' not found in the function declaration [-Wdocumentation]
     @param allowTapToDismiss
            ^~~~~~~~~~~~~~~~~
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:343:9: note: did you mean 'tapToDismissEnabled'?
     @param allowTapToDismiss
            ^~~~~~~~~~~~~~~~~
            tapToDismissEnabled
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:362:20: warning: empty paragraph passed to '@param' command [-Wdocumentation]
     @param queueEnabled
     ~~~~~~~~~~~~~~~~~~^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:20:23: warning: unused variable 'viewController' [-Wunused-variable]
        UIViewController *viewController =
                          ^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:76:22: error: assigning to 'NSNumber *__strong' from incompatible type 'double'
                fontSize = 16.0;
                         ^ ~~~~
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:97:21: warning: unused variable 'topPadding' [-Wunused-variable]
                CGFloat topPadding = window.safeAreaInsets.top;
                        ^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:98:21: warning: unused variable 'bottomPadding' [-Wunused-variable]
                CGFloat bottomPadding = window.safeAreaInsets.bottom;
                        ^
    7 warnings and 1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Encountered error while building for device.


Build failed :|
Failed to build for iOS
```

I use fluttertoast only one time in my app in this way:

```dart
Fluttertoast.showToast(
         msg: "URL copied correctly",
         toastLength: Toast.LENGTH_SHORT,
         gravity: ToastGravity.BOTTOM);
```

This his my build machine:

- MacOS Catalina
- Xcode version: 11.6
- Flutter: 1.20.2 stable

**Comment by ahmeddshahidd** [comment]
Facing the same issue.

**Comment by FinlayFeng** [comment]
### **俺也一样(same for me too)**
`Xcode's output:
↳
    In file included from /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:2:
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:328:19: warning: empty paragraph passed to '@param' command
    [-Wdocumentation]
     @param sharedStyle
     ~~~~~~~~~~~~~~~~~^
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:343:25: warning: empty paragraph passed to '@param' command
    [-Wdocumentation]
     @param allowTapToDismiss
     ~~~~~~~~~~~~~~~~~~~~~~~^
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:343:9: warning: parameter 'allowTapToDismiss' not found in the
    function declaration [-Wdocumentation]
     @param allowTapToDismiss
            ^~~~~~~~~~~~~~~~~
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:343:9: note: did you mean 'tapToDismissEnabled'?
     @param allowTapToDismiss
            ^~~~~~~~~~~~~~~~~
            tapToDismissEnabled
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/UIView+Toast.h:362:20: warning: empty paragraph passed to '@param' command
    [-Wdocumentation]
     @param queueEnabled
     ~~~~~~~~~~~~~~~~~~^
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:20:23: warning: unused variable 'viewController'
    [-Wunused-variable]
        UIViewController *viewController =
                          ^
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:76:22: error: assigning to 'NSNumber *__strong' from
    incompatible type 'double'
                fontSize = 16.0;
                         ^ ~~~~
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:97:21: warning: unused variable 'topPadding'
    [-Wunused-variable]
                CGFloat topPadding = window.safeAreaInsets.top;
                        ^
    /Users/fengli/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-7.0.3/ios/Classes/FluttertoastPlugin.m:98:21: warning: unused variable 'bottomPadding'
    [-Wunused-variable]
                CGFloat bottomPadding = window.safeAreaInsets.bottom;
                        ^
    7 warnings and 1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 11 Pro Max.`

**Comment by ponnamkarthik** [comment]
Fixed in `7.0.4`

**Comment by polilluminato** [comment]
I confirm that there's no issue in `fluttertoast v7.0.4` with my system as listed in my previous comment 👍🏼

**Comment by tevfik1** [comment]
I've got the exact same error with fluttertoast: ^7.0.4

**Comment by deepak786** [comment]
Yes, it is resolved in v7.0.4.
